src/main/java/com/minecolonies/coremod/proxy/ClientProxy.java

```diff
@@ -247,6 +246,61 @@ public static void registerModels(@NotNull final ModelRegistryEvent event)
+                new ModelResourceLocation(new ResourceLocation(Constants.MOD_ID,
+                        BlockTimberFrame.BLOCK_NAME + "_" + BlockPlanks.EnumType.BIRCH.getName()), INVENTORY));
+
+        ModelLoader.setCustomModelResourceLocation(Item.getItemFromBlock(ModBlocks.blockTimberFrameSpruce), 0,
+                new ModelResourceLocation(new ResourceLocation(Constants.MOD_ID,
+                        BlockTimberFrame.BLOCK_NAME + "_" + BlockPlanks.EnumType.SPRUCE.getName()), INVENTORY));
+
+        ModelLoader.setCustomModelResourceLocation(Item.getItemFromBlock(ModBlocks.blockTimberFrameJungle), 0,
+                new ModelResourceLocation(new ResourceLocation(Constants.MOD_ID,
+                        BlockTimberFrame.BLOCK_NAME + "_" + BlockPlanks.EnumType.JUNGLE.getName()), INVENTORY));
+
+        ModelLoader.setCustomModelResourceLocation(Item.getItemFromBlock(ModBlocks.blockTimberFrameDarkOak), 0,
+                new ModelResourceLocation(new ResourceLocation(Constants.MOD_ID,
+                        BlockTimberFrame.BLOCK_NAME + "_" + BlockPlanks.EnumType.DARK_OAK.getName()), INVENTORY));
+
+        ModelLoader.setCustomModelResourceLocation(Item.getItemFromBlock(ModBlocks.blockTimberFrameAcacia), 0,
+                new ModelResourceLocation(new ResourceLocation(Constants.MOD_ID,
+                        BlockTimberFrame.BLOCK_NAME + "_" + BlockPlanks.EnumType.ACACIA.getName()), INVENTORY));
+
+        for (final TimberFrameType type : TimberFrameType.values())
+        {
+            ModelLoader.setCustomModelResourceLocation(Item.getItemFromBlock(ModBlocks.blockTimberFrameOak), type.getMetadata(),
+                    new ModelResourceLocation(ModBlocks.blockTimberFrameOak.getRegistryName() + "_" + type.getName(), INVENTORY));
+        }
+        for (final TimberFrameType type : TimberFrameType.values())
+        {
+            ModelLoader.setCustomModelResourceLocation(Item.getItemFromBlock(ModBlocks.blockTimberFrameSpruce), type.getMetadata(),
+                    new ModelResourceLocation(ModBlocks.blockTimberFrameSpruce.getRegistryName() + "_" + type.getName(), INVENTORY));
+        }
+        for (final TimberFrameType type : TimberFrameType.values())
+        {
+            ModelLoader.setCustomModelResourceLocation(Item.getItemFromBlock(ModBlocks.blockTimberFrameBirch), type.getMetadata(),
+                    new ModelResourceLocation(ModBlocks.blockTimberFrameBirch.getRegistryName() + "_" + type.getName(), INVENTORY));
+        }
+        for (final TimberFrameType type : TimberFrameType.values())
+        {
+            ModelLoader.setCustomModelResourceLocation(Item.getItemFromBlock(ModBlocks.blockTimberFrameAcacia), type.getMetadata(),
+                    new ModelResourceLocation(ModBlocks.blockTimberFrameAcacia.getRegistryName() + "_" + type.getName(), INVENTORY));
+        }
+        for (final TimberFrameType type : TimberFrameType.values())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this loop with the one that starts on line 289. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3047)